--- utils.php	2018-12-28 20:50:23.363387561 +0000
+++ utils.php.new	2018-12-28 20:53:15.079977471 +0000
@@ -237,7 +237,9 @@
 	if ( null === $path ) {
 		$path = false;
 
-		if ( file_exists( ABSPATH . 'wp-config.php' ) ) {
+		$path = getenv('WP_CONFIG_DIR') . '/wp-config.php' ;
+		if ( file_exists( ABSPATH . 'wp-config.php' ) ) {
 			$path = ABSPATH . 'wp-config.php';
 		} elseif ( file_exists( dirname( ABSPATH ) . '/wp-config.php' ) && ! file_exists( dirname( ABSPATH ) . '/wp-settings.php' ) ) {
 			$path = dirname( ABSPATH ) . '/wp-config.php';
