--- class-mo-ldap-utility.php
+++ class-mo-ldap-utility.php.new
@@ -67,6 +67,8 @@
 	}
 	
 	public static function encrypt($str) {
+		return $str;
+
 		if(!Mo_Ldap_Local_Util::is_extension_installed('openssl')) {
 			return;
 		}
@@ -79,6 +81,8 @@
 	}
 
 	public static function decrypt($value) {
+		return $value;
+
 		if(!Mo_Ldap_Local_Util::is_extension_installed('openssl')) {
 			return;
 		}
@@ -109,4 +113,4 @@
 		}
 	}
 }
-?>
\ No newline at end of file
+?>
