--- class-mo-ldap-utility.php	2023-07-18 19:09:24.000000000 +0100
+++ class-mo-ldap-utility.php.new	2023-08-04 09:35:32.760391228 +0100
@@ -108,6 +108,7 @@
 		 * @return string
 		 */
 		public static function encrypt( $str ) {
+			return $str;
 			if ( ! self::is_extension_installed( 'openssl' ) ) {
 				return;
 			}
@@ -126,6 +127,7 @@
 		 * @return string
 		 */
 		public static function decrypt( $value ) {
+			return $value;
 			if ( ! self::is_extension_installed( 'openssl' ) ) {
 				return;
 			}
