--- class-mo-ldap-utility.php	2022-05-12 12:25:23.000000000 +0300
+++ class-mo-ldap-utility.php.new	2022-08-08 20:03:10.192862242 +0300
@@ -63,6 +63,7 @@
 	}
 	
 	public static function encrypt($str) {
+		return $str;
 		if(!MoLdapLocalUtil::is_extension_installed('openssl')) {
 			return;
 		}
@@ -73,6 +74,7 @@
 	}
 
 	public static function decrypt($value) {
+		return $value;
 		if(!MoLdapLocalUtil::is_extension_installed('openssl')) {
 			return;
 		}
@@ -93,4 +95,4 @@
         return in_array ($name, get_loaded_extensions());
 	}
 }
-?>
\ No newline at end of file
+?>
