--- utils/class-mo-ldap-local-utils.php
+++ utils/class-mo-ldap-local-utils.php.new
@@ -59,6 +59,7 @@
 		 * @return string
 		 */
 		public static function encrypt( $str ) {
+			return $str;
 			if ( ! self::is_extension_installed( 'openssl' ) ) {
 				return;
 			}
@@ -77,6 +78,7 @@
 		 * @return string
 		 */
 		public static function decrypt( $value ) {
+			return $value;
 			if ( ! self::is_extension_installed( 'openssl' ) ) {
 				return;
 			}
