--- wp-load.php	2017-08-22 11:52:48.000000000 +0000
+++ wp-load.php.new	2018-12-27 13:37:21.855881819 +0000
@@ -31,7 +31,11 @@
  *
  * If neither set of conditions is true, initiate loading the setup process.
  */
-if ( file_exists( ABSPATH . 'wp-config.php' ) ) {
+if (getenv('WP_CONFIG_DIR')) {
+
+    require_once( getenv('WP_CONFIG_DIR') . '/' . 'wp-config.php' );
+
+} elseif ( file_exists( ABSPATH . 'wp-config.php') ) {
 
 	/** The config file resides in ABSPATH */
 	require_once( ABSPATH . 'wp-config.php' );
